app/domain/operations/eligible/eligibility_configuration.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Operations
+  module Eligible
+    # Configurations for the Eligibility
+    class EligibilityConfiguration
+      def key
+        :eligibility
+      end
+
+      def title
+        "Eligibility"
+      end
+
+      def grants
```

Comment:
the overridden version of this class has 2-D array here (and builder makes use of that) - is 1-D array here ever meant to be acceptable?